src/account/helpers.js

```diff
@@ -138,3 +139,28 @@ export const shortAddressPreview = (addr: string, target: number = 20) => {
     ? addr
     : `${addr.slice(0, slice)}...${addr.slice(addr.length - slice)}`;
 };
+
+export const isUpToDateAccount = (account: ?Account) => {
+  if (!account) return true;
+  const { lastSyncDate, currency } = account;
+  const { blockAvgTime } = currency;
+  if (!blockAvgTime) return true;
+  const outdated =
+    Date.now() - (lastSyncDate || 0) >
+    blockAvgTime * 1000 + getEnv("SYNC_OUTDATED_CONSIDERED_DELAY");
```

Comment:
Why `*1000`?
https://github.com/LedgerHQ/ledger-live-common/blob/a60b6766654436699628469b39482c9f8bbb4d9a/src/types/currencies.js#L65

Reply:
Either it's not `*1000` or the comment on the field should be `// in milliseconds`

Reply:
🤔